guides/deployment/aws.mdx

```diff
@@ -18,45 +18,55 @@ The following guide will walk you through every step to deploy Meilisearch in an
 
 After logging into your [AWS Console](https://aws.amazon.com/console), navigate to the "Compute" service. Then go to "EC2", and finally open your "Instances" console.
 
-![Page titled 'Instances'. Text in center of screen: You do not have any instances in this region](/assets/images/aws/01.launch-instances.png)
+<Frame>
```

Comment:
I think you mentioned briefly over a call, but what do we get from using `<Frame>` instead of regular images? Centred images and the possibility of captions? I'm wondering if it's worth it.

Reply:
Yeah, it just look nicer visually and as you said we got the caption possiblity